test/cypress/integration/05-user_v2.test.js

```diff
@@ -1,26 +1,38 @@
 describe('New users profiles', () => {
   describe('Contributions section', () => {
-    it.skip('Shows contributions with date since and amount contributed', () => {
-      // TODO
+    before(() => {
+      cy.visit('/xdamman/v2');
+      cy.get('[data-cy=Contributions]');
+    });
+    it('Shows contributions with date since and amount contributed', () => {
+      cy.get('[data-cy=collectives]').should('have.length.greaterThan', 1);
+      cy.get('[data-cy=date-since]').contains('since');
+      cy.get('[data-cy=amount-contributed]');
     });
 
-    it.skip('Can load more', () => {
+    it('Can load more', () => {
       // TODO
     });
 
-    it.skip('Can filter by contribution type (admin, financial...etc)', () => {
-      // TODO
+    it('Can filter by contribution type (admin, financial...etc)', () => {
+      cy.get('button[data-cy=filters]').contains(/(Core|Financial|Events)/);
```

Comment:
We do need to test the actual feature by changing the filters to ensure that the items displayed are filtered properly.